tests/utils/test_parameters.py

```diff
@@ -52,44 +53,42 @@ def test_update_empty_dictionary(self):
         d = {"porosity": np.ones(self.g.num_cells)}
         dictionary_kw_list = [d]
         self.p.update_dictionaries(keyword_kw_list, dictionary_kw_list)
-        self.assertIn("porosity", self.p["flow"])
+        assert "porosity" in self.p["flow"]
```

Comment:
IMO, this illustrates how much more elegant pytest is from a Python point of view.